<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>403</width>
    <height>455</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: qlineargradient(spread:pad, x1:0.051, y1:0.954545, x2:0.79, y2:0.744045, stop:0 rgba(190, 0, 192, 160), stop:0.9375 rgba(0, 219, 222, 235))</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">background-color: 
qlineargradient(spread:pad, x1:0.097, y1:0.119, x2:0.978, y2:0.966136, stop:0.0965909 rgba(252, 0, 255, 237), stop:0.9375 rgba(0, 219, 222, 235));</string>
   </property>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>397</width>
      <height>421</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QLineEdit" name="line_login">
      <property name="geometry">
       <rect>
        <x>108</x>
        <y>150</y>
        <width>193</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(0, 0, 0, 0);
border-radius: 2;
border: 1 solid #d3d3d3;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>108</x>
        <y>120</y>
        <width>49</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>10</pointsize>
        <weight>62</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(0 , 0,  0, 0);
color: rgba(255, 255, 255, 240);
font-weight: 500;</string>
      </property>
      <property name="text">
       <string>Login</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>161</x>
        <y>70</y>
        <width>16</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>12</pointsize>
        <weight>62</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(0 , 0,  0, 0);
color: rgba(255, 255, 255, 240);
font-weight: 500;</string>
      </property>
      <property name="text">
       <string>/</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="line_password">
      <property name="geometry">
       <rect>
        <x>108</x>
        <y>240</y>
        <width>193</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(0, 0, 0, 0);
border-radius: 2;
border: 1 solid #d3d3d3;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>108</x>
        <y>210</y>
        <width>73</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>10</pointsize>
        <weight>62</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(0 , 0,  0, 0);
color: rgba(255, 255, 255, 240);
font-weight: 500;</string>
      </property>
      <property name="text">
       <string>Password</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_login">
      <property name="geometry">
       <rect>
        <x>108</x>
        <y>320</y>
        <width>193</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>10</pointsize>
        <weight>62</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(0, 0, 0, 0);
border: 1px solid rgba(255, 255, 255, 255);
color: rgba(255, 255, 255, 240);
font-weight: 500;
border-radius: 15;</string>
      </property>
      <property name="text">
       <string>Sign in</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_signin">
      <property name="geometry">
       <rect>
        <x>108</x>
        <y>70</y>
        <width>52</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>12</pointsize>
        <weight>62</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(0 , 0,  0, 0);
color: rgba(255, 255, 255, 240);
font-weight: 500;
border: none;</string>
      </property>
      <property name="text">
       <string>Sign in</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_signup">
      <property name="geometry">
       <rect>
        <x>168</x>
        <y>70</y>
        <width>61</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>12</pointsize>
        <weight>62</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(0 , 0,  0, 0);
color: rgba(255, 255, 255, 240);
font-weight: 500;
border: none;</string>
      </property>
      <property name="text">
       <string>Sign up</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QLineEdit" name="line_login_registration">
      <property name="geometry">
       <rect>
        <x>108</x>
        <y>135</y>
        <width>193</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(0, 0, 0, 0);
border-radius: 2;
border: 1 solid #d3d3d3;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>161</x>
        <y>70</y>
        <width>16</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>12</pointsize>
        <weight>62</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(0 , 0,  0, 0);
color: rgba(255, 255, 255, 240);
font-weight: 500;</string>
      </property>
      <property name="text">
       <string>/</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_register">
      <property name="geometry">
       <rect>
        <x>108</x>
        <y>320</y>
        <width>193</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>10</pointsize>
        <weight>62</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(0, 0, 0, 0);
border: 1px solid rgba(255, 255, 255, 255);
color: rgba(255, 255, 255, 240);
font-weight: 500;
border-radius: 15;</string>
      </property>
      <property name="text">
       <string>Sign up</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="line_password_registration">
      <property name="geometry">
       <rect>
        <x>108</x>
        <y>195</y>
        <width>193</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(0, 0, 0, 0);
border-radius: 2;
border: 1 solid #d3d3d3;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>108</x>
        <y>170</y>
        <width>73</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>10</pointsize>
        <weight>62</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(0 , 0,  0, 0);
color: rgba(255, 255, 255, 240);
font-weight: 500;</string>
      </property>
      <property name="text">
       <string>Password</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>108</x>
        <y>110</y>
        <width>49</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>10</pointsize>
        <weight>62</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(0 , 0,  0, 0);
color: rgba(255, 255, 255, 240);
font-weight: 500;</string>
      </property>
      <property name="text">
       <string>Login</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="line_name">
      <property name="geometry">
       <rect>
        <x>108</x>
        <y>255</y>
        <width>193</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(0, 0, 0, 0);
border-radius: 2;
border: 1 solid #d3d3d3;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>108</x>
        <y>230</y>
        <width>73</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>10</pointsize>
        <weight>62</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(0 , 0,  0, 0);
color: rgba(255, 255, 255, 240);
font-weight: 500;</string>
      </property>
      <property name="text">
       <string>Name</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_signup_2">
      <property name="geometry">
       <rect>
        <x>168</x>
        <y>70</y>
        <width>61</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>12</pointsize>
        <weight>62</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(0 , 0,  0, 0);
color: rgba(255, 255, 255, 240);
font-weight: 500;
border: none;</string>
      </property>
      <property name="text">
       <string>Sign up</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_signin_2">
      <property name="geometry">
       <rect>
        <x>108</x>
        <y>70</y>
        <width>52</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>12</pointsize>
        <weight>62</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(0 , 0,  0, 0);
color: rgba(255, 255, 255, 240);
font-weight: 500;
border: none;</string>
      </property>
      <property name="text">
       <string>Sign in</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>403</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
